sdk/storage/storage-blob/samples/typescript/basic.ts

```diff
@@ -19,7 +19,7 @@ async function main() {
   const sharedKeyCredential = new SharedKeyCredential(account, accountKey);
 
   // Use TokenCredential with OAuth token
```

Comment:
>TokenCredential [](start = 9, length = 15)

typo `RawTokenCredential`

Reply:
Thanks!  I'll fix the other instances too.